scripts/initialize-celo.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+set -euo pipefail
+
+KEEP_CORE_PATH_DEFAULT=$(realpath -m $(dirname $0)/../../keep-core)
+CONFIG_DIR_PATH_DEFAULT=$(realpath -m $(dirname $0)/../configs)
+KEEP_ECDSA_PATH=$(realpath $(dirname $0)/../)
+KEEP_ECDSA_SOL_PATH=$(realpath $KEEP_ECDSA_PATH/solidity)
+
+# Defaults, can be overwritten by env variables/input parameters
+KEEP_CELO_PASSWORD=${KEEP_CELO_PASSWORD:-"password"}
+NETWORK_DEFAULT="local"
+CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY:-""}
+
+help()
+{
+   echo -e "\nUsage: ENV_VAR(S) $0"\
+           "--keep-ecdsa-config-path <path>"\
+           "--application-address <address>"\
+           "--network <network>"
+   echo -e "\nEnvironment variables:\n"
+   echo -e "\tCONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Celo"
+   echo -e "\nCommand line arguments:\n"
+   echo -e "\t--keep-ecdsa-config-path : Path to keep-ecdsa client configuration file(s)"
```

Comment:
Maybe just `--config-dir`? We use `--config-dir` for the same purpose in `keep-ecdsa/scripts/start.sh` and in `keep-core/scripts/install.sh`.

Reply:
e7e1fd6